deploy/kubernetes/base/controller.yaml

```diff
@@ -91,7 +91,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-attacher
-          image: k8s.gcr.io/sig-storage/csi-attacher:v3.0.0
+          image: k8s.gcr.io/sig-storage/csi-attacher:v3.1.0
```

Comment:
Why we need to update the image version?

Reply:
i didn't update it manually, so i'm assuming one of the `make` recipes bumped the version. @wongma7 do you know why it was updated?

Reply:
yeah probably make generate-kustomize. I double checked and this change makes the helm chart values.yaml match the kustomize, so it's a sound change